cli_tools/common/imagefile/qemu_img.go

```diff
@@ -39,6 +42,7 @@ type ImageInfo struct {
 	Format           string
 	ActualSizeBytes  int64
 	VirtualSizeBytes int64
+	Checksum         string
```

Comment:
Can you update the doc for `ImageInfo` to explain checksum? (The current doc refers only to qemu-img info)